misc/python/materialize/cli/mzconduct.py

```diff
@@ -862,6 +862,25 @@ def __init__(
         )
 
 
+@Steps.register("chaos-kill-container")
```

Comment:
This is fine as is, but you could also implement this as `chaos-kill-service` (or have that as a separate step class) that takes just a service name instead of a container name and does something awful like `find_service_ports` does -- or or just assume that the container name looks like `{comp.name}_{container}_1` -- to get the containers for the service.

Things are slightly more robust as they are written here, it's just that most things in mzcompose/mzconduct operate on services rather than containers, so it's a little unusual to need to know the name of the container. I think it would be fine to just document this as needing the container name and explaining the fact that it differs from what's common.

Mostly this comment is just to point out directions that this can go in the future if figuring container names out gets annoying in the future.

Reply:
I went with the hardcoded `{comp.name}_{container}_1` for now -- thank you for pointing this out! Will probably need to do something like `find_service_ports` in the near future.